admin_ui/backend/api/outbound.py

```
@@ -0,0 +1,771 @@
+    except KeyError:
+        raise HTTPException(status_code=404, detail="Campaign not found")
+    return {"media_uri": media_uri, "campaign": campaign}
+
+
+@router.get("/campaigns/{campaign_id}/voicemail/preview.wav")
+async def preview_voicemail_wav(campaign_id: str):
+    store = _get_outbound_store()
+    try:
+        campaign = await store.get_campaign(campaign_id)
+    except KeyError:
+        raise HTTPException(status_code=404, detail="Campaign not found")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)